packages/node-sdk/src/client.ts

```diff
@@ -976,16 +1069,22 @@ export class BoundBucketClient {
    * @param context User/company/other context to bind to the client object
    * @returns new client bound with the additional context
    */
-  public bindClient({ user, company, other }: Context) {
+  public bindClient({
+    user,
+    company,
+    other,
+    enableTracking = true,
```

Comment:
I think in this case where you're creating `BucketBoundClient` from an existing `BucketBoundClient` we should probably default to inheriting the setting that you have for `enableTracking` and not default to true. The situation i have in mind is that you want to be able to set a basic `BucketBoundClient` with `enableTracking=false` and then at a later point in the middleware series just use whatever setting you had for `enableTracking` further up in the chain.

In short, let's make `enableTracking` optional here like you had previously. Then line 1084 below also makes more sense: use what was given, otherwise use the "parent" `BucketBoundClient`'s value.

Reply:
true